src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/ItemTooltip.java

```diff
@@ -392,6 +394,15 @@ private static Text getMotesMessage(int price, int count) {
         return message;
     }
 
+    private static void smoothenLines(List<Text> lines) {
+        for (int i = 0; i < lines.size(); i++) {
+            Text line = lines.get(i);
+            if (line.getString().equals("-----------------")) {
+                lines.set(i, Text.literal("                      ").formatted(Formatting.DARK_GRAY, Formatting.STRIKETHROUGH, Formatting.BOLD));
```

Comment:
20 spaces is closer.
```suggestion
                lines.set(i, Text.literal("                    ").formatted(Formatting.DARK_GRAY, Formatting.STRIKETHROUGH, Formatting.BOLD));
```